docs/windows/policy/tailscale.admx

```diff
@@ -0,0 +1,256 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<policyDefinitions revision="1.0" schemaVersion="1.0"
+                   xmlns="http://www.microsoft.com/GroupPolicy/PolicyDefinitions">
+  <policyNamespaces>
+    <target prefix="tailscale" namespace="http://www.tailscale.com/GroupPolicy/Tailscale" />
```

Comment:
Even though `http://www.tailscale.com/GroupPolicy/Tailscale` is a valid `xs:anyURI` as per the XML schema definition, the `namespace` attribute should not contain any colons (`:`). Otherwise, the ADMX would not work properly.

In fact, I believe this is the reason why the Requirements Filters aren't working properly with the Tailscale product, and nothing is showing up when the Tailscale checkbox is checked.

As a general rule, the value of the `namespace` attribute should be in the format of a dot-qualified namespace, following the convention used in languages like C# and Java, where different levels are separated by dots. Having just `Tailscale` or `Tailscale.Policies` as the namespace should be perfectly fine for the time being.

Reply:
This fixed the filter!